src/FunFair.CodeAnalysis.Tests/FunFair.CodeAnalysis.Tests.csproj

```diff
@@ -16,6 +16,7 @@
     <CodeAnalysisRuleSet>$(SolutionDir)\CodeAnalysis.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="FunFair.Common.Data" Version="2.4.3.1016-feature-query-arbitr" />
```

Comment:
Don't Not sure we should be importing this... although its a test, its a circular reference.